ce/ce/fs/acquire.ts

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should this be lifted to `ce/fs/acquire.ts`?

Reply:
No, there are multiple projects in the original mono-repo, you can't bring the whole ce up one level, since then it'll mix with the others